deploy/files/prod-public-assets/maintenance.html

```diff
@@ -60,7 +60,7 @@ <h1>We&rsquo;ll be back soon!</h1>
 	<br>
 	<p>Follow us on <a href="https://twitter.com/trustroots">Twitter</a> and <a href="https://www.facebook.com/trustroots.org">Facebook</a></p>
 	<br>
-	<p>Check our <a href="http://status.trustroots.org/">status page</a></p>
+	<p>Check our <a href="https://status.trustroots.org/">status page</a></p>
```

Comment:
This one doesn't seem to function properly. In any case https is better than http. Review and fixing of links can be done in separate PR.